src/main/RP.ts

```diff
@@ -1,18 +1,22 @@
+import { DomainLinkageCredential, IVerifyCredentialResult, WellKnownDidVerifier } from '@sphereon/wellknown-dids-client';
+import WDCErrors from '@sphereon/wellknown-dids-client/dist/constants/Errors';
```

Comment:
Hmzz, apparently this isn't being exported properly. We need to fix that.

Reply:
fixed